crates/chia-bls/src/bls_cache.rs

```diff
@@ -138,26 +145,19 @@ impl BlsCache {
         use pyo3::types::PyBytes;
         let ret = PyList::empty_bound(py);
         for (key, value) in &self.cache {
-            ret.append((
-                PyBytes::new_bound(py, key),
-                PyBytes::new_bound(py, &value.to_bytes()),
-            ))?;
+            ret.append((PyBytes::new_bound(py, key), value.clone().into_py(py)))?;
```

Comment:
we talked about going back to using plan `bytes` here, since `GTElement` doesn't support pickle. (maybe it should in the future, but let's make that a separate PR)

Reply:
Fixed